Dockerfile

```diff
@@ -34,35 +37,33 @@ RUN dnf -y module enable python38
 # of dependencies below. For example, drpm-devel.
-RUN dnf -y install python38 python38-cryptography python38-devel
-RUN dnf -y install openssl openssl-devel
-RUN dnf -y install openldap-devel
-RUN dnf -y install wget git
-RUN dnf -y install python3-psycopg2
-RUN dnf -y install redhat-rpm-config gcc cargo libffi-devel
-RUN dnf -y install glibc-langpack-en
-RUN dnf -y install python3-libmodulemd
-RUN dnf -y install python3-libcomps
-RUN dnf -y install libpq-devel
-RUN dnf -y install python3-setuptools
-RUN dnf -y install swig
-RUN dnf -y install buildah --exclude container-selinux
-RUN dnf -y install xz
-RUN dnf -y install libmodulemd-devel
-RUN dnf -y install libcomps-devel
-RUN dnf -y install zchunk-devel
-RUN dnf -y install ninja-build
-RUN dnf -y install cairo-devel cmake gobject-introspection-devel cairo-gobject-devel
-RUN dnf -y install libcurl-devel libxml2-devel sqlite-devel #file-devel
-RUN dnf -y install ostree-libs #ostree
-RUN dnf -y install skopeo
-RUN dnf -y install podman
-RUN dnf -y install sudo
-RUN dnf -y install zstd
-RUN dnf -y install doxygen
-RUN dnf -y install graphviz # the dot command
-RUN dnf -y install file-devel
-RUN dnf clean all
+RUN dnf -y install python38 python38-cryptography python38-devel && \
+    dnf -y --setopt=install_weak_deps=False install openssl openssl-devel && \
+    dnf -y --setopt=install_weak_deps=False install openldap-devel && \
+    dnf -y --setopt=install_weak_deps=False install wget git && \
+    dnf -y --setopt=install_weak_deps=False install python3-psycopg2 && \
+    dnf -y --setopt=install_weak_deps=False install python3-createrepo_c && \
+    dnf -y --setopt=install_weak_deps=False install redhat-rpm-config gcc cargo libffi-devel && \
+    # dnf -y install gcc cargo libffi-devel && \
+    dnf -y --setopt=install_weak_deps=False install glibc-langpack-en && \
+    # dnf -y install python3-libmodulemd && \
+    # dnf -y install python3-libcomps && \
+    dnf -y --setopt=install_weak_deps=False install libpq-devel && \
+    dnf -y --setopt=install_weak_deps=False install python3-setuptools && \
+    # dnf -y install swig && \
+    # dnf -y install buildah --exclude container-selinux && \
+    dnf -y --setopt=install_weak_deps=False install xz && \
+    dnf -y --setopt=install_weak_deps=False install libmodulemd-devel && \
+    dnf -y --setopt=install_weak_deps=False install libcomps-devel && \
+    dnf -y --setopt=install_weak_deps=False install zchunk-devel && \
+    # dnf -y install ninja-build && \
+    dnf -y --setopt=install_weak_deps=False install cmake gobject-introspection-devel cairo-gobject-devel && \
+    # dnf -y install cairo-devel cmake gobject-introspection-devel cairo-gobject-devel && \
+    # dnf -y install libcurl-devel libxml2-devel sqlite-devel file-devel && \
+    dnf -y --setopt=install_weak_deps=False install libcurl-devel sqlite-devel file-devel && \
+    dnf -y --setopt=install_weak_deps=False install ostree-libs ostree && \
```

Comment:
I think it is sufficient to install just one of `ostree-libs` and `ostree`. I am not sure which one is required for the python bindings.

Reply:
Gonna remove each and see what's gonna happen. :crossed_fingers:

Reply:
Also, after yesterday's meetings, I am sure we can remove everything related to ostree for now. @dkliban, do you agree?